app/models/concerns/dossier_champs_concern.rb

```diff
@@ -177,6 +177,22 @@ def reload
     super.tap { reset_champs_cache }
   end
 
+  # FIXME: this is a temporary fix to remove duplicated champs with the same nil row_id
+  def tmp_fix_uniq_row_ids
+    duplicated_champ_ids = champs.where(row_id: [Champ::NULL_ROW_ID, nil])
+      .order(id: :desc)
+      .select(:id, :stream, :stable_id, :row_id)
+      .group_by { "#{_1.stream}-#{_1.public_id}" }
+      .values
+      .flat_map { |(_, *duplicated_champs)| duplicated_champs.map(&:id) }
```

Comment:
suis-je le seul a trouvé ça difficile à lire ? une option qui me parle un peu plus :
```suggestion
      .flat_map { |duplicated_champs| duplicated_champs[1..] }
```

autrement dit, les duplicats sont tous ceux après le 1er element ?

Reply:
Oui. Je garde le plus récent. Je n'ai pas mieux comme heuristique simple. Si cela te semble trop cavalier, je veux bien en parler. Il y a aussi la possibilité, dans un premier temps, de déplacer les champs dans un stream `bad_data` plutôt que de supprimer.